tests/e2e/utils/requestHandlers/headers/CheMultiuserAuthorizationHeaderHandler.ts

```diff
@@ -22,6 +23,6 @@ export class CheMultiuserAuthorizationHeaderHandler implements IAuthorizationHea
 
     async get(): Promise<AxiosRequestConfig> {
         const token = await this.tokenHandler.get();
-        return { headers: { 'Authorization': `Bearer ${token}` } };
+        return { headers: { 'Authorization': `Bearer ${token}` }, httpsAgent: new https.Agent({ rejectUnauthorized: false }) };
```

Comment:
Why do you prefer new https.Agent({ rejectUnauthorized: false }) instead of NODE_TLS_REJECT_UNAUTHORIZED=0?

Reply:
I was not aware of this. So this is an environment variable that should be set before test run? cc @ScrewTSW

Reply:
It's not necessarily that I would prefer it. I honestly didn't try using the variable.
This approach just worked for me and it was like a third thing that I tried, so I was satisfied ^^;
We could try to test it by changing the env variable and if it works I will revert my commit.

Reply:
Note please:
In case with variable we have 2 benefits:
1) The var operates globally on node process and we do not need add  new https.Agent({ rejectUnauthorized: false }) construction 
2) With variable all classes that use axios requests will be work correct. In current case it just resolve problem with TLS in the CheMultiuserAuthorizationHeaderHandler.ts

Reply:
I removed all occurrences of `https.Agent` from the code.